src/google/protobuf/compiler/csharp/csharp_options.h

```diff
@@ -0,0 +1,60 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
This needs to be added into `protobuf/cmake/extract_includes.bat.in` and `src/Makefile.am`, I believe - I think that's why the Travis build is failing.